public/index.html

```diff
@@ -31,16 +88,16 @@ <h2 class="shop-item__title">T-Shirts</h2>
                 <span class="fa fa-square"></span>
 
                 <ul class="dropdown__options">
-                  <li class="dropdown__options-item shirt-color--pink">
+                  <li class="dropdown__options-item-shirt-color--pink">
```

Comment:
I think this is undoing some of your work and leading to code duplication. `dropdown__options-item` should do all of the styling for a single item in the dropdown (borders, size, etc) then the `shirt-color--***` just changes the font color